### PR TITLE
sudo: 1.9.5p2 -> 1.9.6

### PR DESCRIPTION
**Pull request by NeQuissimus**
###### Motivation for this change
Update

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [X] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 116423` at c18933f4 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package skipped due to time constraints:</summary>
  <ul>
    <li>mysql-workbench</li>
  </ul>
</details>
<details>
  <summary>7 packages built successfully:</summary>
  <ul>
    <li><a href="https://gist.github.com/5bea081b30fc9fc173793054ae1332a9">charliecloud</a></li></li>
    <li><a href="https://gist.github.com/ed224f1a035f6e2e2c41b301fb2ed8a4">interlock</a></li></li>
    <li><a href="https://gist.github.com/4aed3840e4bb0b30d8ac76b183e93bc8">lxqt.lxqt-sudo</a></li></li>
    <li><a href="https://gist.github.com/9617264e7231d1747db8c0d0dbbd13bf">passExtensions.pass-tomb</a></li></li>
    <li><a href="https://gist.github.com/6d40c702daa32bfd427afd4cff6e2216">qsudo</a></li></li>
    <li><a href="https://gist.github.com/584c9c40586d87e72f46311dc925c63b">sudo</a></li></li>
    <li><a href="https://gist.github.com/7607d6361297cc0b2e795aacb7d1dd00">tomb</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>error: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-flags-array.md">no-flags-array</a>

`configureFlagsArray` is only intended to be used in Bash, not as a Nix attribute.

Near pkgs/tools/security/sudo/default.nix:45:3:
```
   |
45 |   configureFlagsArray = [
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/tools/security/sudo/default.nix [hunk 1]
```diff
@@ -58,7 +65,7 @@ stdenv.mkDerivation rec {
 
   postInstall =
     ''
-    rm -f $out/share/doc/sudo/ChangeLog
+      rm -f $out/share/doc/sudo/ChangeLog
```
Comment (SuperSandro2000) [inline pkgs/tools/security/sudo/default.nix:68]:
```suggestion
      rm $out/share/doc/sudo/ChangeLog
```
we want to notice when the file is missing.

File: pkgs/tools/security/sudo/default.nix [hunk 2]
```diff
@@ -58,7 +65,7 @@ stdenv.mkDerivation rec {
 
   postInstall =
     ''
```
Comment (SuperSandro2000) [inline pkgs/tools/security/sudo/default.nix:67]:
```suggestion
  postInstall = ''
```
when we are reformatting we should fix this, too.

**Review by delroth — APPROVED** [review]
Thanks for the package bump and cleanup!

**Pull request merged**